tests/Makefile.am

```diff
@@ -9,7 +9,8 @@ check_PROGRAMS =
 TESTS_ENVIRONMENT = \
   abs_top_testdir=`cd '$(top_srcdir)'/tests; pwd` \
   abs_top_builddir=`cd '$(top_builddir)'; pwd` \
-  abs_top_srcdir=`cd '$(top_srcdir)'; pwd`
+  abs_top_srcdir=`cd '$(top_srcdir)'; pwd` \
+  UNINSTALLED=1
```

Comment:
I would prefer to not having to set UNINSTALLED=1 for every test I run from the command line. I think your test running should set INSTALLED=1.

Reply:
ok